frontend/lib/src/components/shared/Icon/styled-components.ts

```diff
@@ -60,7 +60,7 @@ export const StyledIcon = styled("span", {
     isPropValid(prop) && !["size", "as"].includes(prop),
 })<StyledIconProps>(({ color, size, margin, padding, theme }) => {
   return {
-    color: theme.colors[color],
```

Comment:
Changes this to directly apply the color instead of trying to match it in our theme.